omc3/scripts/mqt_extraction.py

```diff
@@ -0,0 +1,79 @@
+"""
+MQT Extraction
+---------------
+
+Extract MQT (Quadrupole Trim) knob values from NXCALS for a specific time and beam.
+
+**Arguments:**
+
+*--Required--*
+
+- **time** *(datetime|str)*:
+
+    The timestamp for which to retrieve the data (timezone-aware recommended). If a string is provided, it will be parsed to a datetime object, assuming UTC timezone.
+
+- **beam** *(int)*:
+
+    The beam number (1 or 2).
+
+*--Optional--*
+
+- **output_path** *(str|Path)*:
+    Path to the output file or directory. If a directory is given, the file will be named 'extracted_mqts.str'. If not provided, no output will be written.
+"""
```

Comment:
I wonder if we should have `mqt_extraction` and `knob_extractor` not follow the same naming scheme and be in the same parent folder in `omc3`